### PR TITLE
ci(bug): Update permissions in JSON rpc relay for GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by rbarkerSL**
**Description**:
Added `packages: write` permission per [the github documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic) May need to modify other files modified by #3124

**Related Issue(s)**:
Relates to #3124
Relates to #3182

### CODE REVIEW

**Review by ebadiere — APPROVED** [review]
LGTM

**Review by quiet-node — APPROVED** [review]

File: .github/workflows/release-production.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ env:
 
 permissions:
   contents: read
+  packages: write
```
Comment (Nana-EC) [inline .github/workflows/release-production.yml:14]: Q: Shouldn't this be the only change?
Not sure why there are version changes of dependencies included in the PR also.
Can those be removed?

**Review by Nana-EC — COMMENTED** [review]
Change seems right but additional dependency changes seem out of place.

**Review by quiet-node — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]
LG

**Pull request merged**